### ISSUE TITLE
Fix travis tests/builds failing due to timeout

### ISSUE THREAD

**Issue by davidlmobley** [open-forcefield-group/smarty#228]
Nightly builds and our most recent PR are failing because of issues like this on Travis: https://travis-ci.org/open-forcefield-group/smarty/jobs/211486127

It gets to a message listing a big block of packages to be installed then has something like this:
```
Proceed ([y]/n)? 
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
```
I've seen this several times recently. It's unclear to me if Travis-CI just adjusted its timeout settings (so we just need to adjust our settings) or if something has changed that is making things stall or run slower than normal. We'll need to investigate.

**Comment by bannanc** [comment]
@davidlmobley is there something I can do to get this fixed sooner rather than later? It would be good to get this fixed before pull request #234 is ready for final review.

**Comment by jchodera** [comment]
I just fixed this in #182

**Comment by jchodera** [comment]
The issue is discussed here: https://github.com/omnia-md/conda-recipes/pull/739 https://github.com/conda/conda-build/issues/1825

**Comment by jchodera** [comment]
Here's the current workaround for future reference: https://github.com/open-forcefield-group/smarty/pull/182/commits/46d7c8314c3f1f31f26b8f85d24e7e85a2d054e3

**Comment by davidlmobley** [comment]
I just merged #182 so I'm marking this as closed. Thanks, @jchodera .
